EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+
+In other words, we need to measure the ETH "real" value. This EIP proposes to allocate *much* (it probably may be more than 90%) of miner's ETH fees to Future Salaries contract. So the cost of ETH will be associated with the real ("job") value of the conditional tokens of this contract.
+
+So, this proposal partly solves both the well-known problems of
+- decreasing ETH supply
+- making ETH cost proportionally increasing when tokens' value increases (in other word incresing the ETH intristic value)
+
+As the above mentioned RFC explains, it is good for miners/validators, because:
+- It reduces manipulation incentives such as selfish mining
+- It highly increases the security of the Ethereum network
+- It increases the ETH intristic value, so miners/validators spend less on energy per amount of money earned.
+- There is the danger (https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0001-positioning/0001-positioning.md) that without radical measures like this proposal the value of ETH (not of Ethereum tokens) will become zero in some future, so greatly downshifting miners/validators.
+
+## Specification
+1. Finish the smart contract audit (it was already paid for) of [SalaryWithDAO](https://github.com/vporton/future-contracts/blob/master/contracts/SalaryWithDAO.sol) and [DefaultDAOInterface](https://github.com/vporton/future-contracts/blob/master/contracts/DefaultDAOInterface.sol) contracts.
+2. After the audit is finished, deploy these contracts (together with a DAOstack DAO controlling the `DefaultDAOInterface` with voting tokens initially allocated by a community consensus) and create an oracle ID using the `SalaryWithDAO` API.
+3. Audit and deploy [DonateETH](https://github.com/vporton/donations/blob/main/contracts/DonateETH.sol) redirecting ETH payments to this oracle, too.
+4. Discuss the fractions of miners fees and transfer fees of ETH to be transferred to `DonateETH` at every mining and transfer operation.
+5. Change the Ethereum clients to do these ETH transfers.
+
+In the future (it should be a separate EIP) we can switch to some more sophisticated DeFi scheme (`SalaryWithDAO` contract does support donations in DeFi token) of converting ETH to ERC-1155 tokens to be transferred to `SalaryWithDAO` than simply wrapping ETH into ERC-1155 token by `DonateETH`, to increase future value of the tokens transferred to `SalaryWithDAO`.
+
+Additional proposal for future consideration:
+
+- Consider (by the DAO voting) "obliging" the above mentioned DAO by a smart contract (with ownership transfer) to set the minimum oracle finish date in the future to some fixed value (need to discuss the exact number of years to lock funds for).
+- Consider (in a separate EIP) creating new oracles with shifted to father future (e.g. by one year forward once per year) the minimum finish times and spreading the transferred funds between several such oracles.
+
+## Rationale
```

Comment:
@MicahZoltu Please explain what it means "mint ETH into this DAO". First, the DAO just controls account restoration and nothing other. So minting ETH to DAO makes no sense. Apparently, you mean that you want me to explain why not to send ETH directly to `SalaryWithDAO` contract? It makes no sense because `SalaryWithDAO` may have multiple "oracles" and it is not possible to determine which oracle to donate to. So we need `DonateETH` "wrapper". Is it your question?

What do you mean by "be balanced with spending"? Do you mean my statement that ETH needs to represent not only its own value but also values of tokens?

Reply:
It is possible I am misunderstanding the proposal at the moment.  I'll wait until the specification is complete to comment further on this particular topic.

Reply:
Oh, it seems I understood you: Do you mean why I propose to have the amount of ETH transferred to Future Salaries to be a fixed fraction of the miner's fees?

You are right, there is no strong reason for this. It would be more energy efficient to transfer ETH there in big chunks. I will fix the specification.

Reply:
Corrected.

Reply:
> It is possible I am misunderstanding the proposal at the moment. I'll wait until the specification is complete to comment further on this particular topic.

I've edited. Please comment.

Reply:
I still don't fully understand why we can't just mint some ETH for this.  That would result on a tax on all holders, rather than a tax on users.